app/boards/arm/mikoto/mikoto.yaml

```
@@ -1,5 +1,5 @@
-identifier: mikoto_520
-name: mikoto_520
+identifier: mikoto
+name: mikoto
```

```suggestion
name: Mikoto 5.20
```
In theory, you could change this so the display name in the script is "Mikoto 5.20". Then you could add copies of this file with `identifier: mikoto@6.1.0` `identifier: mikoto@7.1.0` with the corresponding `name`s, for instance. However I am not sure if everything in ZMK scripts would work well with the `@` syntax or something would break, I guess we could try and see.

---

I'll give that a go